tools/docker-dev/Dockerfile

```diff
@@ -26,40 +27,44 @@ vim \
 
 # Create user and allow sudo without password.
 RUN addgroup --quiet --gid $gid $group \
-&& adduser --quiet --disabled-password --gecos ',,,,' --uid $uid --ingroup $group $user \
+&& adduser --quiet --disabled-password --gecos ",,,," --uid $uid --ingroup $group $user \
 && echo "${user} ALL=(ALL:ALL) NOPASSWD: ALL" > /etc/sudoers.d/$user
 
 # Install Docker CLI.
 COPY --from=docker /usr/local/bin/docker /usr/local/bin/docker
 
 # Fix the Docker socket access rights at login time to allow non-root access.
-RUN echo 'sudo chmod o+rw /var/run/docker.sock' >> /home/${user}/.bashrc
+RUN echo "sudo chmod o+rw /var/run/docker.sock" >> /home/${user}/.bashrc
 
 # Install Go.
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
This is breaking the following `RUN`. It seems like it prevents the rewriting of `${goversion}`.

Reply:
Actually it's broken because the `ca-certificates` package seems too old. `curl` can't find the root CA for `dl.google.com`. That's because you changed from `ubuntu:latest` to `ubuntu:xenial`.

Reply:
Installing `ca-certificates=20170717~16.04.2` fixes that.